pinot-controller/src/main/java/org/apache/pinot/controller/helix/ControllerRequestURLBuilder.java

```diff
@@ -18,6 +18,8 @@
  */
 package org.apache.pinot.controller.helix;
 
+import com.google.common.base.Joiner;
+import java.util.List;
 import javax.annotation.Nullable;
 import org.apache.commons.lang.StringUtils;
```

Comment:
Not from this PR, but let's change it to `lang3` for consistency
```suggestion
import org.apache.commons.lang3.StringUtils;
```